packages/rest/src/lib/RequestManager.ts

```diff
@@ -269,7 +257,12 @@ export class RequestManager extends EventEmitter {
 
 				// Fire event
 				this.emit(RESTEvents.HashSweep, sweptHashes);
-			}, this.options.hashSweepInterval).unref();
+			}, this.options.hashSweepInterval);
+
+			// Only unref in node envs
+			if (typeof this.hashTimer !== 'number') {
+				this.hashTimer = this.hashTimer.unref();
```

Comment:
No need to reassign here, just call unref. Almost makes sense to just .unref?.() instead.